tests/integration/targets/s3_object/tasks/get_object_info.yml

```diff
@@ -0,0 +1,128 @@
+---
+- name: Integration test for s3_object_info
+  block:
+    - name: Create a simple s3_bucket
+      s3_bucket:
+        name: "{{ bucket_name }}-object-info-test"
+        state: present
+        object_ownership: BucketOwnerEnforced
+        tags:
+          "lowercase spaced": "hello cruel world"
+          "Title Case": "Hello Cruel World"
+          CamelCase: "SimpleCamelCase"
+          snake_case: "simple_snake_case"
+
+    - name: check that temp directory was made
+      assert:
+        that:
+        - remote_tmp_dir is defined
+
+    - name: Create content
+      set_fact:
+          content: "{{ lookup('password', '/dev/null chars=ascii_letters,digits,hexdigits,punctuation') }}"
+
+    - name: Create local object-info-test.txt
+      copy:
+        content: "{{ content }}"
+        dest: "{{ remote_tmp_dir }}/object-info-test.txt"
+
+    - name: stat the file
+      stat:
+        path: "{{ remote_tmp_dir }}/object-info-test.txt"
+        get_checksum: yes
+
+    - name: Upload a object to bucket
+      amazon.aws.aws_s3:
+        bucket: "{{ bucket_name }}-object-info-test"
+        object: object-info-test.txt
+        src: "{{ remote_tmp_dir }}/object-info-test.txt"
+        mode: put
+        permission: bucket-owner-full-control
+        tags:
+          "lowercase spaced": "hello cruel world"
+          "Title Case": "Hello Cruel World"
+          CamelCase: "SimpleCamelCase"
+          snake_case: "simple_snake_case"
+
+    - name: List keys
+      amazon.aws.s3_object_info:
+        bucket_name: "{{ bucket_name }}-object-info-test"
+      register: list_keys_result
+
+    - name: Get s3 object info
+      amazon.aws.s3_object_info:
```

Comment:
There is a lot of redundancy with `s3_object` tests. We end up spending a lot of time just to prepare/teardown the set-up.

Can we just call these `s3_object_info` tasks somewhere during `s3_object` tests and remove this extra test target?